code/game/machinery/computer/supply.dm

```diff
@@ -65,6 +65,7 @@
 		dat += {"
 		<A href='?src=\ref[src];viewrequests=1'>View requests</A><BR>
 		<A href='?src=\ref[src];vieworders=1'>View orders</A>"}
+	dat += "</html>"
```

Comment:
This should be hoisted into the else because otherwise, if(temp) you'll end up with an extra </html>